pgscatalog_utils/ancestry/ancestry_analysis.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import textwrap
+import logging
+import glob
+import os
+
+import pandas as pd
+
+import pgscatalog_utils.config as config
+from pgscatalog_utils.ancestry.read import read_pcs, read_pgs, extract_ref_psam_cols
+from pgscatalog_utils.ancestry.tools import assign_ancestry, _assign_method_threshold, choose_pval_threshold, \
+    pgs_adjust, _normalization_methods
+
+logger = logging.getLogger(__name__)
+
+
+def ancestry_analysis():
+    args = _parse_args()
+    config.set_logging_level(args.verbose)
+    config.OUTDIR = args.outdir
+
+    # Load PCA data
+    maxPCs = max([10, args.nPCs_assignment, args.nPCs_normalization])  # save memory by not using all PCs
+    loc_ref_pcs = glob.glob('*{}*.pcs'.format(args.d_ref))
+    reference_df = read_pcs(loc_pcs=loc_ref_pcs, dataset=args.d_ref,
+                            loc_related_ids=args.ref_related, nPCs=maxPCs)
+    loc_ref_psam = glob.glob('GRCh38_{}_ALL.psam'.format(args.d_ref))[0]
```

Comment:
will this break if reference data are `GRCh37`?

Reply:
Oops, that wasn't meant to be hardcoded. Can switch to `'*{}*.psam'.format(args.d_ref)`